neuralprophet/components/router.py

```diff
@@ -0,0 +1,69 @@
+from neuralprophet.components.trend.linear import GlobalLinearTrend, LocalLinearTrend
+from neuralprophet.components.trend.piecewise_linear import GlobalPiecewiseLinearTrend, LocalPiecewiseLinearTrend
+from neuralprophet.components.trend.static import StaticTrend
+
+
+def get_trend(config, id_list, quantiles, num_trends_modelled, n_forecasts, device):
+    """
+    Router for all trend classes.
+
+    Based on the conditions provided, the correct trend class is returned and initialized using the provided args.
+    """
+    if config.growth == "off":
+        # No trend
+        return StaticTrend(
```

Comment:
Please complete docstrings.

We could also parse the function args into a dict and then parse(`**`) that into the appropriate trend classes (as all the args seem to be identical and identically named)